integration/jca/src/main/java/org/apache/cxf/jca/core/classloader/PlugInClassLoader.java

```diff
@@ -95,7 +95,7 @@ private static Properties getProperties(ClassLoader parent, String propsFileName
 
         props.load(in);
         LOG.fine("Contents: " + propsFileName + props);
-
+        in.close();
```

Comment:
Could we wrap it in `try(InputStream in = parent.getResourceAsStream(propsFileName)) {...}` please?

Reply:
Needed to rewrite a bit of this in order to do this, as try-with-resources does not allow a stream that is set in try-with-resources to be reset later within the try.      Split it out into two try-with-resources statements.